lib/rules/jsx-curly-spacing.js

```diff
@@ -30,34 +32,87 @@ module.exports = {
     fixable: 'code',
 
     schema: [{
-      enum: SPACING_VALUES
```

Comment:
this seems like it's removing the old schema, which makes it a breaking change - the idea would be to allow all existing configuration possibilities to continue to work, but provide an alternative that people could migrate to.

Specifically, this also means that (separate from bugfixes) zero existing tests should be modified, and should all continue to pass.